_blueprints/developers_guide.adoc

```diff
@@ -3,15 +3,16 @@
 == Adding new packages
 
 ALMighty uses https://glide.sh[Glide] for package management.  To
-install a new package, you can follow these steps.  The `<URL>` should
-be replaced with the package URL pointing to Github or any other
-location.
+install a new package, you can follow these steps.  The
+`<IMPORT_PATH>` should be replaced with the actual import path of the
```

Comment:
`<IMPORT_PATH>` is not better because it could mean import to `vendor/github.com/foo/bar`. See [this](Package paths) page and search for "Package Path" to read about it. It should be `<PACKAGE_PATH>`.